btnclick.htm

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>KIDS LEARNING APP</title>
+    
+    <!--adding css style to-->
+    <link rel="stylesheet" href="styles/css/btn-click.css" />
+
+    <!--linking js file via defer-->
+    <script src="js/btnclick.js" defer></script>
+  </head>
+
+
+  <!--body container-->
+  <body>
+    <!--parent-container-->
+    <article class="parent_container">
+      <!--header container-->
+      <section class="header">
```

Comment:
use of semantic tags

Reply:
semantic tags has been used instead of the div tags